pkg/v1/cli/command/core/plugin_manager.go

```diff
@@ -222,21 +231,43 @@ var describePluginCmd = &cobra.Command{
 var installPluginCmd = &cobra.Command{
 	Use:   "install [name]",
 	Short: "Install a plugin",
```

Comment:
```suggestion
	Short: "Install a plugin",
	Args:  cobra.ExactArgs(1),
```

instead of the explicit check at line 226